tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -410,6 +418,8 @@ else if (args[i].equals("-D") || args[i].equals("-d"))
            doDebugging        = !doDebugging;
       else if (args[i].equals("-STAT") || args[i].equals("-stat")) 
            doStats      = !doStats;      
+      else if (args[i].equals("-error-codes"))
```

Comment:
I suggest to call `toLowerCase` on `args[i]` to ignore casing.

Reply:
Done